### ISSUE TITLE
Cannot install SharePoint Yeoman Generator

### ISSUE THREAD

**Issue by johnggreen** [SharePoint/sp-dev-docs#862]
#### Category
- [ ] Question
- [ ] Typo
- [X] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
Expected behaviour is that the yeoman generator installs successfully. 

#### Observed Behavior
Has the git repository been moved? I've followed the guides on [https://dev.office.com/sharepoint/docs/spfx/toolchain/scaffolding-projects-using-yeoman-sharepoint-generator](https://dev.office.com)

24 error git clone git@github.com:microsoft/generator-sharepoint Cloning into bare repository 
24 error git clone git@github.com:microsoft/generator-sharepoint Host key verification failed.
24 error git clone git@github.com:microsoft/generator-sharepoint fatal: Could not read from remote repository

#### Steps to Reproduce
> Install the latest version of Node JS x64 for Windows
> Open command line and run 
`npm install @microsoft/generator-sharepoint -g`

#### Attached Files - Logs for 
[npm-debug.zip](https://github.com/SharePoint/sp-dev-docs/files/1311542/npm-debug.zip)

**Comment by waldekmastykarz** [comment]
Could you try using Node LTS (v6) with npm@3 and see if you still get the error? Your log mentions you're using Node v8 with npm@1, whereas SPFx has been tested with Node LTS.

Also, it's quite odd that your system tries to pull the generator from GitHub instead of npmjs. Did you change any configuration related to packages registry?

**Comment by johnggreen** [comment]
Hi, thanks for your quick response. I ran it again with the Node LTS v6 installed and experienced the same issue. Looking at the logs however, I noticed that the following lines were referencing an old  chocolatey path for npm which wasn't desired.  This appeared to be some left over configuration from a previous install. 
'npm verb cli [ 'C:\\Program Files\\nodejs\\node.exe',
npm verb cli   'C:\\ProgramData\\chocolatey\\lib\\npm\\tools\\node_modules\\npm\\bin\\npm-cli.js',
'
Once I uninstalled this version, I was able to run the below successfully
'npm install @microsoft/generator-sharepoint -g' 

I tried re-running the command against Node v8.5, npm 5.3.0 and it worked correctly as described, so it looks like this issue was caused by some incorrect configuration on my machine. 

Thanks for your assistance anyway!
